content/en/docs/KRE/user/40_kais_runner_sdk/_index.md

```diff
@@ -24,10 +117,52 @@ description: >
 
 ## Context
 
+The context object provides a set of utilities as database client or logger that can be used
+for different purposes on `init` or `handler` functions.
+
+The context provides all these functionalities:
+
+- **In-Memory Storage**: A volatile variable storage that will be available from start until the version is stopped or restarted. You can set a value with a key name and retrieve it later by that key.
+
+- **Logging**: Functions to register any message or error with different log levels: `debug, info, warning, error`.
+
+- **Data Persistence**: A way to persist and/or retrieve data from a DB.
+
+- **Measurements**: An easy way to save any amount of arbitrary measurements you need to use later on an Influx graph.
+
+- **Metrics**: Allows you to save predicted and real data in order to feed the pre-generated charts on the "Metrics" menu for each version.
+
 ### Database
 
+During any part of your workflows you may need to persist data or recover previously saved data from a DB.  
+In those cases you can use the following functions:
+
+```golang
+    func Find(colName string, query QueryData, res interface{}) error 
+    func Save(colName string, data interface{}) error
+    
+    // QueryData is the following type:
+    type QueryData map[string]interface{} 
+```
+
 ### Logger
 
+You can use four log levels that will be shown on Admin UI page for the Version it belongs. The SDK will take care of saving the timestamp and tag each log with metadata to locate the source of the error.
```

Comment:
on "the" Admin UI
Could be nice to also add "with metadata to help locate" to the sentence